overlay.py

```diff
@@ -243,6 +244,7 @@ def get_overlay_args(overlay_description: str):
             action="store",
             type=str,
             choices=["v2", "V2", "v3", "V3"],
+            default=configs["bike"],
```

Comment:
If `configs["bike"]` is the default, and line 45 also directs to `configs["bike"]`, then we probably don't need to set a default value for this argument, if you get what I mean.

Reply:
Good catch! I've instead removed the default on line 45, since I reckon it's more useful to be able to see what the default is when we use `--help`.